pkg/awsv1alpha1/accounts.go

```diff
@@ -31,11 +31,11 @@ func GetAccountCRsToReset(ctx context.Context, cli client.Client) ([]awsv1alpha1
 }
 
 // SetAccountStateReady sets an account state to Ready
-func SetAccountStateReady(cli client.Client, account awsv1alpha1.Account) error {
-	account.Status.State = "Ready"
+func ResetAccountStatus(cli client.Client, account awsv1alpha1.Account) error {
+	account.Status = awsv1alpha1.AccountStatus{}
```

Comment:
My only concern here is that if an account is failing for some reason and constantly being reset it will just eat cycles and prevent other accounts from being shredded or created in a timely manner, as I feel it would create a cycle like:

Account Reconciles -> Fails -> Shredded -> Status Reset -> Reconciles in the Account Operator -> Fails -> Shredded -> Status reset etc.

Reply:
Right, this is why we didn't reset the status at all when we create the shredder. But the current desire is for the accounts to be automatically reset. Ready is not the right choice for that.

Reply:
@iamkirkbater One though, we could try to identify any accounts that get in that hot loop with metrics